Architect-Coders/MU8
# Issue #48: Use Navigation Component

**What's in:**

- Updated version for ship stats service to v3.

**Testing Done:**

* On real device: Pixel 2 XL.

**How it looks:**

![MU8NavComponent](https://user-images.githubusercontent.com/3532556/73141670-ae10f980-4054-11ea-8963-cf478f5ab767.jpg)

- Video for Navigation: https://drive.google.com/open?id=1T64U_OgyheqQbrTCzD1JqoZDTtHCR-TT

**How to test:**

- Launch the app and navigate through it.
- Check the new navigation using the navigation components from JetPack.
- Check that there is no error on any of the screens already implemented.

**Signed-off-by:**

* Diego Recalde <difereto89@gmail.com>
